sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/OpenAIServerSentEvents.java

```diff
@@ -15,11 +15,13 @@
 import java.nio.ByteBuffer;
 import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 public final class OpenAIServerSentEvents<T> {
 
-    private static final String STREAM_COMPLETION_EVENT = "data: [DONE]";
+    private static final List<String> STREAM_COMPLETION_EVENT = Arrays.asList("data: [DONE]", "data:[DONE]");
```

Comment:
Is this a bug?

Reply:
not really. The service team changed for a bit the format of the messages and they got rid of the blank space. I added it just in case that happens again in a future deployment.